edison/static/js/policy.controller.js

```diff
@@ -155,12 +175,19 @@ function initSettingToNewPolicy() {
     policy.reset();
 }
 
+
 function addPolicy() {
```

Comment:
Ok, I will think about this

Reply:
Just to be clear, this is the appropriate way to handle this - `this function should add the policy in the client side also`.

Reply:
OK, but it should be only added to the client if the sending to the backend succeed,right ?

Reply:
Yes, but you can already implement it.
Send an ajax call within this function to `/policy` with the appropriate body (look at the API we designed) and in the 'success' callback add the new policy.

Reply:
OK, but I need the token for this implementation, right?

Reply:
The token can be added in the future when the JWT PR will be approved.
When it gets approved if you won't add the token the ajax error callback will be fired and the policy won't be added (because it's being added in the success callback).
You can add a comment stating that once the JWT mechanism implemented, a token should be added to this ajax call.

Reply:
OK, I will do it in the future.